hack/util.sh

```diff
@@ -71,21 +71,14 @@ sti::log::error_exit() {
   exit "${code}"
 }
 
-os::util::sed() {
+s2i::util::sed() {
```

Comment:
I'd opt for consistency with the rest?
We don't do a great job at keeping this stuff in sync (humans...), and in most cases it is not a problem, the projects can diverge.